<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Menu</class>
 <widget class="QDialog" name="Menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>619</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">theShellMenu</string>
  </property>
  <property name="windowIcon">
   <iconset theme="theshell" resource="resources.qrc">
    <normaloff>:/icons/icon.svg</normaloff>:/icons/icon.svg</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="AnimatedStackedWidget" name="stackedWidget">
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="InstallLayout">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="font">
             <font>
              <pointsize>20</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Install theShell OS</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Give theShell OS a permanant place on your PC. Install theOS on your hard drive, either replacing another operating system or next to another operating system.</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_3">
              <property name="text">
               <string>Open theShell OS Installer</string>
              </property>
              <property name="icon">
               <iconset theme="install_theos">
                <normaloff>.</normaloff>.</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <pointsize>20</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Gateway</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListView" name="appsListView">
           <property name="contextMenuPolicy">
            <enum>Qt::CustomContextMenu</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit">
           <property name="inputMethodHints">
            <set>Qt::ImhNoPredictiveText</set>
           </property>
           <property name="frame">
            <bool>true</bool>
           </property>
           <property name="placeholderText">
            <string>Start typing to search, run a command, or open a web address</string>
           </property>
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string notr="true">username</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="reportBugButton">
             <property name="toolTip">
              <string>Report Bug</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset theme="tools-report-bug">
               <normaloff>.</normaloff>.</iconset>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="helpButton">
             <property name="toolTip">
              <string>Help</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset theme="help-contents">
               <normaloff>.</normaloff>.</iconset>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton">
             <property name="text">
              <string>End Session</string>
             </property>
             <property name="icon">
              <iconset theme="system-shutdown">
               <normaloff>.</normaloff>.</iconset>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="text">
          <string>End Session</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="shutdownText">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>You're about to power off your PC. Are you sure?</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_22">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QCommandLinkButton" name="exitButton">
           <property name="text">
            <string>Exit theShell</string>
           </property>
           <property name="icon">
            <iconset theme="application-exit">
             <normaloff>.</normaloff>.</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
           <property name="description">
            <string>Exits theShell, leaving everything else open</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCommandLinkButton" name="fakeEndButton">
           <property name="text">
            <string>Fake Exit</string>
           </property>
           <property name="iconSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCommandLinkButton" name="commandLinkButton">
           <property name="styleSheet">
            <string notr="true">background-color: #A00;
color: #FFF;</string>
           </property>
           <property name="text">
            <string>Power Off</string>
           </property>
           <property name="icon">
            <iconset theme="system-shutdown">
             <normaloff>.</normaloff>.</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
           <property name="description">
            <string>Turns off your computer</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCommandLinkButton" name="commandLinkButton_2">
           <property name="styleSheet">
            <string notr="true">background-color: #A00;
color: #FFF;</string>
           </property>
           <property name="text">
            <string>Reboot</string>
           </property>
           <property name="icon">
            <iconset theme="system-reboot">
             <normaloff>.</normaloff>.</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
           <property name="description">
            <string>Turns off your computer and turns it back on again</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCommandLinkButton" name="commandLinkButton_3">
           <property name="text">
            <string>Log Out</string>
           </property>
           <property name="icon">
            <iconset theme="system-log-out">
             <normaloff>.</normaloff>.</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
           <property name="description">
            <string>Ends your session and keeps the computer on for other people to use</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCommandLinkButton" name="commandLinkButton_5">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Suspend</string>
           </property>
           <property name="icon">
            <iconset theme="system-suspend">
             <normaloff>.</normaloff>.</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
           <property name="description">
            <string>Puts your computer in a low power state so that it opens up quickly</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCommandLinkButton" name="commandLinkButton_6">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Lock</string>
           </property>
           <property name="icon">
            <iconset theme="system-lock-screen">
             <normaloff>.</normaloff>.</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
           <property name="description">
            <string>Locks your workspace with your password</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCommandLinkButton" name="commandLinkButton_8">
           <property name="text">
            <string>Turn Off Screen</string>
           </property>
           <property name="icon">
            <iconset theme="video-display">
             <normaloff>.</normaloff>.</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
           <property name="description">
            <string>Turns off the screen, but keeps everything else running in the background</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCommandLinkButton" name="commandLinkButton_4">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Switch Users</string>
           </property>
           <property name="icon">
            <iconset theme="system-switch-user">
             <normaloff>.</normaloff>.</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
           <property name="description">
            <string>Switches to the logon screen and keeps your session in the background</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCommandLinkButton" name="commandLinkButton_7">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Hibernate</string>
           </property>
           <property name="icon">
            <iconset theme="system-suspend-hibernate">
             <normaloff>.</normaloff>.</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
           <property name="description">
            <string>Powers off your computer, but when powered back on, restores the session.</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="text">
            <string>Cancel</string>
           </property>
           <property name="icon">
            <iconset theme="dialog-cancel">
             <normaloff>.</normaloff>.</iconset>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_3">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="text">
          <string>End Session</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Before you end your session, here are some things you might want to know</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="endSessionWarnings">
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="verticalScrollMode">
          <enum>QAbstractItemView::ScrollPerPixel</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QPushButton" name="cancelEndSessionWarningButton">
           <property name="text">
            <string>Cancel</string>
           </property>
           <property name="icon">
            <iconset theme="dialog-cancel">
             <normaloff>.</normaloff>.</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="endSessionAnywayButton">
           <property name="text">
            <string notr="true">Power off anyway</string>
           </property>
           <property name="icon">
            <iconset theme="system-shutdown">
             <normaloff>.</normaloff>.</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_4">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="font">
          <font>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Switch Users</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Who are you switching to?</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="availableUsersList"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QPushButton" name="cancelSwitchUsersButton">
           <property name="text">
            <string>Cancel</string>
           </property>
           <property name="icon">
            <iconset theme="dialog-cancel">
             <normaloff>.</normaloff>.</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="startNewSessionButton">
           <property name="text">
            <string>Start a new session</string>
           </property>
           <property name="icon">
            <iconset theme="system-switch-user">
             <normaloff>.</normaloff>.</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_5">
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_10">
         <property name="font">
          <font>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Failed to start app</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="startupFailLabel">
         <property name="text">
          <string notr="true">xyz can't start</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AnimatedStackedWidget</class>
   <extends>QStackedWidget</extends>
   <header>animatedstackedwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
